modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -2005,6 +2005,53 @@ TEST_F(InferWithReshapeNV12, TestInferListYUV)
     // Validate
     validate();
 }
+
+#if INF_ENGINE_RELEASE >= 2020010000
```

Comment:
Why such `#if` needed in there, if it's handled inside `giewrapper`

Reply:
It is for findDataFile(). For age gender we can find path for old IE. I don't know old path for semantic segmentation. And old IE isn't tested now, old path for network is deprecated.

old path - look at `std::string SUBDIR` *

Reply:
Should I delete it?

Reply:
Removed test.